source/_posts/2016-04-25-opencomponents-microservices-in-the-front-end-world.markdown

```diff
@@ -0,0 +1,259 @@
+---
+layout: post
+title: "OpenComponents - microservices in the front-end world"
+date: 2016-04-25 10:10:03 +0000
+comments: true
+author: mfigus
+categories: [SOA, Microsites, Microservices, OpenComponents, OC] 
+---
+
+Many engineers work every day on Opentable.com from our offices located in Europe, America, and Asia, pushing changes to production multiple times a day. Usually, this is very hard to achieve, in fact it took years for us to get to this point. I explained in a [previous article][1] how we dismantled our monolith in favour of a Microsites architecture. In the meanwhile, during the last two years we have been working on something I would define quite unique, given its purpose and its definition, called **OpenComponents**.
```

Comment:
I explained in a [previous article][1] how we dismantled our monolith in favour of a Microsites architecture. Since the publication of that blog post we have been working on something I would define as quite unique called **OpenComponents**.

^^^ This reads better for me IMO. But that's just me..